ArjunaJTS/jtax/classes/com/arjuna/ats/internal/jta/recovery/jts/jca/coordinator/RecoveredServerTransaction.java

```diff
@@ -0,0 +1,487 @@
+/*
```

Comment:
Ah right, so there is a diff:

> import com.arjuna.ats.internal.jta.transaction.jts.subordinate.jca.coordinator.ServerTransaction;
56d44
< import com.arjuna.ats.internal.jts.orbspecific.interposition.coordinator.ServerTransaction;

So the only way to avoid the duplicate code would be to refactor ServerTransaction which I agree that that is something we should avoid in this particular PR.